blocks/layout-grid/style.scss

```diff
@@ -186,16 +186,22 @@
  * Individual column alignment
  */
 .wp-block-jetpack-layout-grid-column {
+	display: flex;
+	height: 100%;
```

Comment:
This height rule likely makes the editor-only height rule redundant, [this one](https://github.com/Automattic/block-experiments/pull/252/files#diff-479fde507612e22d5a3adf773cece3ab696340ff1f19b7ab771056be352b5de3R42). I still kept it for the time being.